core/app/presenters/spree/csv/product_variant_presenter.rb

```diff
@@ -88,9 +88,9 @@ def call
           variant.backorderable?,
           variant.tax_category&.name,
           variant.digital?,
-          variant.images[0]&.attachment&.url,
-          variant.images[1]&.attachment&.url,
-          variant.images[2]&.attachment&.url,
+          variant.images[0]&.original_url,
```

Comment:
The `original_url` calls the `cdn_url`: https://github.com/spree/spree/blob/main/core/app/models/concerns/spree/image_methods.rb#L21